sheeprl/algos/dreamer_v3/agent.py

```diff
@@ -68,14 +71,12 @@ def __init__(
                 input_channels=self.input_dim[0],
                 hidden_channels=(torch.tensor([2**i for i in range(stages)]) * channels_multiplier).tolist(),
                 cnn_layer=nn.Conv2d,
-                layer_args={"kernel_size": 4, "stride": 2, "padding": 1, "bias": not layer_norm},
+                layer_args={"kernel_size": 4, "stride": 2, "padding": 1, "bias": layer_norm_cls == nn.Identity},
                 activation=activation,
-                norm_layer=[LayerNormChannelLast for _ in range(stages)] if layer_norm else None,
-                norm_args=(
-                    [{"normalized_shape": (2**i) * channels_multiplier, "eps": 1e-3} for i in range(stages)]
-                    if layer_norm
-                    else None
-                ),
+                norm_layer=[layer_norm_cls] * stages,
+                norm_args=[
```

Comment:
Shouldn't we pop out the `normalized_shape` kwarg from the `layer_norm_kwargs` to prevent that the user sets it from the configs?

Reply:
It is not necessary in this case: when composing a python dictionary in this way, the keys you specify after the `**kwargs` rewrite the keys in `kwargs`. You must pop the key in case you are calling a function or initializing an object (as done in the LayerNormGruCell.__init__` function (otherwise an error is raised).